src/forms/subscriptions.ts

```diff
@@ -1,19 +1,15 @@
-import asyncBatch from 'async-batch/lib';
-
-import {Channel} from 'mattermost-redux/types/channels';
-import {AppSelectOption, AppCallRequest, AppForm, AppField} from 'mattermost-redux/types/apps';
-
 import {AppFieldTypes} from 'mattermost-redux/constants/apps';
+import {AppCallRequest, AppForm, AppField, AppSelectOption} from 'mattermost-redux/types/apps';
```

Comment:
In general it is easier if one sorts these references/imports alphabetically (easier to search for one later) - inside an import statement and the import statement themselves.

Reply:
This will be addressed with issue #77. There will likely be many changes throughout the codebase and I'd like to address those separately.  I've added this comment as a request for that issue.